packages/permission-controller/src/PermissionController.test.ts

```diff
@@ -5337,4 +5400,209 @@ describe('PermissionController', () => {
       expect(error).toMatchObject(expect.objectContaining(expectedError));
     });
   });
+
+  describe('permission groups', () => {
+    it('are properly handled when a single permission group is granted', () => {
```

Comment:
Tests should be written as a readable sentence, e.g.:

```suggestion
    it('grants the child permissions in a permissions group', () => {
```

"it are properly [...]" sounds strange. 😅

Reply:
lol i was struggling to come up with something that sounded sane, thank you